backend/llm/private_gpt4all.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         self,
         model: str,
         chat_id: str,
-        brain_id:str,
+        brain_id: str,
         temperature: float,
         max_tokens: int,
         user_openai_api_key: str,
```

Comment:
Same question, do we need them here too when already specified in BaseBrainPIcking ?

Reply:
I Think this is on an old commit?